### DISCUSSION TITLE
RA and DEC telescope control issue

### DISCUSSION THREAD

**Discussion by ak-team** [Stellarium/stellarium#1606] [Q&A] [answered]
Hi,

I am exporting the RA and DEC values from the telescope control plugin using a python script.

I have a question followed by an issue. Let me start with the question:

RA and DEC values of nearby objects like the sun, moon, satellites and planets change while that of stars do not. Can someone kindly explain why?  I will post the issue after your answers as maybe I have an understanding gap. I thought RA and DEC changes negligibly over the night? 

Thanks a lot in advance :)

Best Regards,

**Comment by gzotti** [comment, marked as answer]
> 
> 
> Hi,
> 
> I am exporting the RA and DEC values from the telescope control plugin using a python script.
> 
> I have a question followed by an issue. Let me start with the question:
> 
> RA and DEC values of nearby objects like the sun, moon, satellites and planets change while that of stars do not. Can someone kindly explain why? 

Maybe because planets are moving in the sky between the fixed stars?

**Reply by ak-team** [reply]
So, you mean since the planets are closer, the angle changes faster than stars which are much further away and hence the extreme slow change rates ? In short, closer objects will have faster changing RA and DEC values than objects further away? Thanks :)

**Reply by gzotti** [reply]
Generally, yes. Sideways motion of 15 km/s in 800.000.000 km distance is more apparent than motion of 150 km/s in 5.000.000.000.000.000 km distance.
You may want to find some general introductory information to astronomy online or even in our User Guide.

**Reply by ak-team** [reply]
Thanks a lot for explaining everything :) Your reply confirmed a concept which helped me finish a tiny code I was working on. Issue resolved and 1 step closer to enjoying the night sky :)
